Formula/heimdal.rb

```diff
@@ -24,8 +24,7 @@ class Heimdal < Formula
   keg_only "conflicts with Kerberos"
 
   depends_on "bison" => :build
-  depends_on "berkeley-db"
-  depends_on "flex"
+  depends_on "flex" => :build
```

Comment:
Linkage is broken, so this is needed at runtime.

Reply:
Ah, I had the feeling this was actually needed. Any suggested comments to clarify this?

Reply:
`# required at runtime`?